cli/login_test.go

```diff
@@ -142,6 +194,12 @@ func TestLogin(t *testing.T) {
 			"password", coderdtest.FirstUserParams.Password,
 			"password", coderdtest.FirstUserParams.Password, // confirm
 			"trial", "yes",
+			"firstName", coderdtest.TrialUserParams.FirstName,
+			"lastName", coderdtest.TrialUserParams.LastName,
+			"phoneNumber", coderdtest.TrialUserParams.PhoneNumber,
+			"jobTitle", coderdtest.TrialUserParams.JobTitle,
+			"companyName", coderdtest.TrialUserParams.CompanyName,
```

Comment:
If trial is not set, then these parameters should not be required.

Reply:
yeah that's right, but in this https://github.com/coder/coder/blob/10ccce1bd99210095895c27b21f04dda8d90cc30/cli/login_test.go#L196 trial is already set, so the params are required and I have added another test case `InitialUserTTYWithNoTrial` where trial is not set and params are not required
https://github.com/coder/coder/blob/10ccce1bd99210095895c27b21f04dda8d90cc30/cli/login_test.go#L128

Reply:
Oops, I must have missed that! Thanks for clarifying.